modules/rpc/thrift.go

```diff
@@ -36,27 +35,21 @@ var _setupMu sync.Mutex
 type CreateThriftServiceFunc func(svc service.Host) ([]transport.Procedure, error)
 
 // ThriftModule creates a Thrift Module from a service func
-func ThriftModule(hookup CreateThriftServiceFunc, options ...modules.Option) service.ModuleCreateFunc {
```

Comment:
How the code is in this PR barely changes usage. If anything, I like the idea of being able to say:

```
var FooThriftModuleFunc = rpc.ThriftModule(FooThriftServiceFunc)

WithModule("myFoo", FooThriftModuleFunc, mySpecificOptionsForThisRegistration...)
```